lib/facter/docker.rb

```diff
@@ -67,3 +67,24 @@ def interfaces
     end
   end
 end
+
+Facter.add(:systemroot) do
```

Comment:
I'm not keen on these fact names.  It's very possible they will collide with other modules that need to do same thing.

Perhaps prefix with `docker_` or something?

Also it's generally considered better practice for one file per fact. Although I see prior art says otherwise.